part-spec/power.json

```diff
@@ -111,44 +177,410 @@
                     "description": "describes integrated fet current limits and rdson properties",
                     "$ref": "#/$defs/powerFetProperities"
                 },
-                "undervoltageLockoutThresholdRising": {
-                    "description": "Undervoltage Lockout Out (UVLO) Threshold with input voltage rising",
-                    "comment": "units of Volts",
+                "componentProtectionThresholds": {
+                    "description": "Thermal and power supply protection thresholds of a device",
+                    "type": "array",
+                    "items": {
+                        "$ref": "definitions.json#/componentProtectionThresholds"
+                    }
+                },
+                "efficiency": {
+                    "description": "power efficiency of regulator",
+                    "$ref": "graph.json#/graphDefiniton"
+                },
+                "pins": {
+                    "description": "array of pin objects with associated properties",
+                    "type": "array",
+                    "items": {
+                        "$ref": "definitions.json#/pinSpec"
+                    }
+                },
+                "package": {
+                    "description": "component's package size and description",
+                    "items": {
+                        "$ref": "definitions.json#/package"
+                    }
+                }
+            }
+        },
+        {
+             "title": "ldo",
+              "type": "object",
+              "required": [
+                "componentID",
+                "vinMin",
+                "vinMax",
+                "voutMin",
+                "voutMax",
+                "maxLoadCurrent"
+              ],
+          "properties": {
```

Comment:
The indentation around here seems off: line 206  (`{`) has 8 spaces, line 207 (`"title"`) has 13 spaces, and this line 217 (`"properties"`) has 10 spaces. The rest of the file seems to use 4 spaces consistently, do you mind fixing this up? (consider just running it through a JSON formatter to avoid tedium :] )

Reply:
Indentation changed